docs/sources/developers/http_api/dashboard.md

```diff
@@ -16,22 +16,511 @@ labels:
 title: Dashboard HTTP API
 ---
 
-# Dashboard API
+# New Dashboard APIs
 
 > If you are running Grafana Enterprise, for some endpoints you'll need to have specific permissions. Refer to [Role-based access control permissions](/docs/grafana/latest/administration/roles-and-permissions/access-control/custom-role-actions-scopes/) for more information.
 
-## Identifier (id) vs unique identifier (uid)
+> To view more about the new api structure, refer to [API overview]({{< ref "apis" >}}).
 
-The identifier (id) of a dashboard is an auto-incrementing numeric value and is only unique per Grafana install.
+## Create Dashboard
 
-The unique identifier (uid) of a dashboard can be used for uniquely identify a dashboard between multiple Grafana installs.
+`POST /apis/dashboard.grafana.app/v1beta1/namespaces/:namespace/dashboards`
+
+Creates a new dashboard.
+
+- namespace: to read more about the namespace to use, see the [API overview]({{< ref "apis" >}}).
+
+**Required permissions**
+
+See note in the [introduction]({{< ref "#dashboard-api" >}}) for an explanation.
+
+<!-- prettier-ignore-start -->
+| Action              | Scope                                                                                                   |
+| ------------------- | ------------------------------------------------------------------------------------------------------- |
+| `dashboards:create` | <ul><li>`folders:*`</li><li>`folders:uid:*`</li></ul>                                                   |
+| `dashboards:write`  | <ul><li>`dashboards:*`</li><li>`dashboards:uid:*`</li><li>`folders:*`</li><li>`folders:uid:*`</li></ul> |
+{ .no-spacing-list }
+<!-- prettier-ignore-end -->
+
+**Example Create Request**:
+
+```http
+POST /apis/dashboard.grafana.app/v1beta1/namespaces/default/dashboards HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+
+{
+  "metadata": {
+    "name": "gdxccn",
+    "annotations": {
+      "grafana.app/folder": "fef30w4jaxla8b"
+    },
+  },
+  "spec": {
+    "title": "New dashboard",
+    "schemaVersion": 41
```

Comment:
🤔 odd because this is actually an invalid dashboard

but a valid one is kinda too big for the example... should we put "..." in the value so it is more obviously partial?

Reply:
👍 good idea, updated

Reply:
Also added one full body